lib/core/AxiosError.js

```diff
@@ -93,6 +101,7 @@ AxiosError.from = (error, code, config, request, response, customProps) => {
   axiosError.name = error.name;
 
   customProps && Object.assign(axiosError, customProps);
+  customProps && (axiosError.customProps = Object.keys(customProps));
```

Comment:
What if we will write these 2 statements with `if(customProps) {...}`?

Reply:
I can totally rewrite theses 2 statements into an if statement.